pyk/src/pyk/kcfg.py

```diff
@@ -197,6 +199,7 @@ def to_dict(self) -> Dict[str, Any]:
         target = sorted(self._target)
         expanded = sorted(self._expanded)
         verified = [{"source": source_id, "target": target_id} for source_id, target_id in sorted(self._verified)]
+        aliases = self._aliases
```

Comment:
```suggestion
        aliases = dict(self._aliases)
```

Reply:
Any particular reason for this? `_aliases` is already a dict.

Reply:
`_aliases` is a mutable collection, so in order protect class invariants we should return a copy and not the original reference.

Reply:
Makes sense, thanks.